cli/src/template/ios/MoproApp.xcodeproj/project.pbxproj

```diff
@@ -42,6 +51,9 @@
 		2A5992322CC7750C00135402 /* plonk_fibonacci_vk.bin */ = {isa = PBXFileReference; lastKnownFileType = archive.macbinary; path = plonk_fibonacci_vk.bin; sourceTree = "<group>"; };
 		2A5992362CC77A7E00135402 /* mopro.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = mopro.swift; sourceTree = "<group>"; };
 		2A5992372CC77A7E00135402 /* MoproBindings.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; path = MoproBindings.xcframework; sourceTree = "<group>"; };
+		3FEF6E9A2DC444C8000C5D5E /* zkemail.json */ = {isa = PBXFileReference; lastKnownFileType = text.json; name = zkemail.json; path = "../init/test-vectors/noir/zkemail.json"; sourceTree = SOURCE_ROOT; };
+		3FEF6E9E2DC444D2000C5D5E /* zkemail_input.json */ = {isa = PBXFileReference; lastKnownFileType = text.json; name = zkemail_input.json; path = "../init/test-vectors/noir/zkemail_input.json"; sourceTree = SOURCE_ROOT; };
+		3FEF6E9F2DC444D2000C5D5E /* zkemail_srs.local */ = {isa = PBXFileReference; lastKnownFileType = file; name = zkemail_srs.local; path = "../init/test-vectors/noir/zkemail_srs.local"; sourceTree = SOURCE_ROOT; };
```

Comment:
you should copy the file into the xcode project 
instead of setting relative path
users will create a `mopro-example-app` anywhere not only relative to `mopro` github path

![](https://github.com/user-attachments/assets/18c09133-1b3b-4d1d-9685-6706904d6fa4)

Reply:
You're right, I selected the wrong property. fixed in [94e9b82](https://github.com/zkmopro/mopro/pull/399/commits/94e9b82be45b5f2310508da875de4e670e95f85a)

Reply:
Yep, you're right, I selected the wrong type. fixed in [3a9dd22](https://github.com/zkmopro/mopro/pull/399/commits/3a9dd22c7160276b3d30cb9f396cda69782350c8)